libs/GraphicECS/SFML/Components/SoundComponent.hpp

```diff
@@ -0,0 +1,44 @@
+/*
+** EPITECH PROJECT, 2022
+** R-Type
+** File description:
+** SoundComponent
+*/
+
+#ifndef SOUNDCOMPONENT_HPP_
+#define SOUNDCOMPONENT_HPP_
+
+#include "Component/Component.hpp"
+#include "GraphicECS/SFML/Resources/SoundResource.hpp"
+
+using namespace graphicECS::SFML::Resources;
+
+namespace graphicECS::SFML::Components
+{
+    /// @brief SoundComponent stores the key as enum to the corresponding sound stored in shared resource
+    /// SoundResource.
+    class SoundComponent : public ecs::Component {
+      public:
+        /// @brief Enumeration of all status available for a Sound.
+        /// Status STOPED specify that a sound is STOP but at its begin
+        enum status_e { STOPED, PAUSED, PLAYING };
```

Comment:
Seams useless, sure it can be possible to stop, pause and play music with sf::Sound ?

Reply:
otherwise, STOPPED instead of STOPED

Reply:
PLAYED

Reply:
it seems possible to play, stop, pause, nevermind